packages/cozy-mespapiers-lib/src/components/Contexts/helpers.js

```diff
@@ -0,0 +1,40 @@
+import { index, addDocs } from '../../utils/search'
+import { FILES_DOCTYPE } from '../../doctypes'
+
+const updateFile = doc => {
+  if (doc.trashed) {
+    index.remove(doc._id)
+  }
+}
+
+export const onUpdate = doctype => async doc => {
+  if (doctype === FILES_DOCTYPE) {
```

Comment:
So, no update for contacts?

Reply:
on ne peut pas éditer un contact depuis l'app, je pense que j'avais oublié le cas par exemple d'un contact édité dans l'app Contacts depuis un autre onglet 👍